projectview/src/main/java/org/jetbrains/bsp/bazel/projectview/parser/splitter/ProjectViewRawSection.java

```diff
@@ -4,16 +4,20 @@
 
 public class ProjectViewRawSection {
 
-  private final String sectionHeader;
+  private final String sectionName;
   private final String sectionBody;
 
-  public ProjectViewRawSection(String sectionHeader, String sectionBody) {
-    this.sectionHeader = sectionHeader;
+  public ProjectViewRawSection(String sectionName, String sectionBody) {
+    this.sectionName = sectionName;
     this.sectionBody = sectionBody;
   }
 
-  public String getSectionHeader() {
-    return sectionHeader;
+  public boolean compareByName(String thatSectionName) {
```

Comment:
Maybe unnecessary indirection. Additionally, if it's named `compareByName`, I'd expect the argument to be a `ProjectViewRawSection`.

Reply:
yeah, it needs to to take `String` so probably a better name would be `.doesHaveName(String)`